lib/gcs.rb

```diff
@@ -48,6 +48,41 @@ def get_object(bucket, object=nil, download_dest: nil)
     end
   end
 
+  def read_partial(bucket, object=nil, limit: 1024*1024, trim_after_last_delimiter: nil, &blk)
+    bucket, object = _ensure_bucket_object(bucket, object)
+    uri = URI("https://www.googleapis.com/download/storage/v1/b/#{bucket}/o/#{CGI.escape(object)}?alt=media")
+    Net::HTTP.start(uri.host, uri.port, use_ssl: true) do |http|
```

Comment:
`require "uri"`, `require "cgi"`, `require "net/http"` are missing, but it works.
(maybe they are loaded in google-api-client and/or its dependent gems)
I prefer to load them explicitly in this file. How do you think?